integration_tests/tests/apply/viewApplication.cy.ts

```diff
@@ -120,28 +123,28 @@ context('show applications', () => {
     AND('I should see a link to the assessment with guidance')
     showPage.shouldShowAssessmentDetails(true)
 
-    WHEN(`I click the 'Request for placement' tab`)
+    WHEN(`I click the 'PLacement request' tab`)
     cy.task('stubApplicationRequestsForPlacement', {
       applicationId: updatedApplication.id,
       requestsForPlacement: [],
     })
-    showPage.clickRequestAPlacementTab()
+    showPage.clickLink('Placement request')
 
-    THEN(`I do not see the 'Create request for placement' button`)
+    THEN(`I do not see the 'Create placement request' button`)
```

Comment:
Given the button now appears before the tabs (if it does appear at all!), this assertion could be moved up -- there should be no need to click the placement request tab at all.